test/unittests/stack_test.cpp

```diff
@@ -136,7 +134,7 @@ TEST(operand_stack, top)
     EXPECT_EQ(stack.top().i64, 101);
     EXPECT_EQ(stack[0].i64, 101);
 
-    stack.shrink(0);
```

Comment:
Hm, this reminds me don't have a test for `stack.drop(0);`, it should be a no op. Perhaps just add here before `drop(1)`?